src/Console/Command/CommandHelp.php

```diff
@@ -269,6 +270,21 @@ public static function toString($value)
         return var_export($value, true);
     }
 
+    public static function getDisplayableAllowedValues(FixerOption $option)
+    {
+        if (null !== $allowed = $option->getAllowedValues()) {
```

Comment:
good practice. too easy to write
`if `($foo = bar()) {}`
then you don't know did one wanted to just assign and check is it truly or compare function result to variable